schema/workflow.json

```diff
@@ -1767,35 +1777,45 @@
       ]
     },
     "schedule": {
-      "type": "object",
-      "description": "Start state schedule definition",
-      "properties": {
-        "interval": {
-          "type": "string",
-          "description": "Time interval (ISO 8601 format) describing when workflow instances can be created."
-        },
-        "cron": {
-          "$ref": "#/definitions/crondef"
-        },
-        "directInvoke": {
-          "description": "Define if workflow instances can be created outside of the defined interval/cron",
-          "type": "boolean",
-          "default": false
-        },
-        "timezone": {
-          "type": "string",
-          "description": "Timezone name used to evaluate the cron expression. Not used for interval as timezone can be specified there directly. If not specified, should default to local machine timezone."
-        }
-      },
       "oneOf": [
         {
-          "required": [
-            "interval"
-          ]
+          "type": "string",
+          "description": "Time interval (ISO 8601 format) describing when workflow instances can be created.",
+          "minLength": 1
         },
         {
-          "required": [
-            "cron"
+          "type": "object",
+          "description": "Start state schedule definition",
+          "properties": {
+            "interval": {
+              "type": "string",
+              "description": "Time interval (ISO 8601 format) describing when workflow instances can be created.",
+              "minLength": 1
+            },
+            "cron": {
+              "$ref": "#/definitions/crondef"
+            },
+            "directInvoke": {
```

Comment:
iirc we added directInvoke specifically when we added cron definition, it was requested (by Kay if I'm not mistaken) as it would indicate if a cron-based workflow could be triggered explicitly outside of the cron definition

Reply:
I'll speak with Kay about that, but I don't think it makes sense to have it at this level.  Really, I question whether we should even have a flag to disable directInvoke at all.  What do you think about completely dropping that?

Since this PR is already merged and I'm late to respond, would you prefer if I send a follow-up PR for the following proposal?

a) remove directInvoke flag (or alternatively move it out of the 'schedule' definition?)
b) Change the examples to use repeating interval like you have in your other comment below
c) Change the description of interval to say that it must be a repeating interval

Reply:
sure, go for it. might want to do after we decide on https://github.com/serverlessworkflow/specification/pull/266 tho as that moves the start definition from workflow states to top-level prop (just saying so you dont have to rebase later in case that one gets merged first).

and +1 for removing directInvoke